lib/tasks/update_supplier_for_moves.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+namespace :moves do
+  desc 'Update the suppliers for all the moves based on the effective dates of the Suppliers'
+  task update_suppliers: :environment do
+    total = Move.count
+
+    i = 0
+
+    puts 'Updating suppliers for moves...'
+    Move.find_each(batch_size: 200) do |move|
+      move.determine_supplier # TODO: this might need to change
+
+      if ENV['confirm_save'] == 'true'
```

Comment:
Not really keen on using environment variables for this - or that we need this extra complexity to be honest. If you want to keep it then would be better to use a parameter passed into the rake task instead - and document it in the rake task description.

Reply:
Yeah I'll remove it and I'll add more useful log